src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(auto-fill,200px);
+  gap: 46px;
+  justify-content: center;
+
+  padding: 50px 0;
+  margin: 0 auto;
```

Comment:
not fixed comment about margins for block.
margin should be set for element
for example
```
.main__catalog {
  margin: 0 auto;
}